packages/dynamite/dynamite_runtime/lib/src/client/client.dart

```diff
@@ -100,12 +99,7 @@ class DynamiteClient with http.BaseClient {
     if (validStatuses?.contains(response.statusCode) ?? true) {
       return response;
     } else {
```

Comment:
This should go away, right? Otherwise we have validation in two places :thinking:

Reply:
We still need it for the raw requests.
The last step once #1707 has landed is going to remove the entire executeRequest and executeaRawRequest methods.

Reply:
My bad. I just double checked and we already use the DynaMiteSerializer in the RequestManager so this could probably be removed.

We'd only need to bring back the functionality for other callers like the RequestManager.wrapUri method.

Should we just leave it for now until it's removed or should I find a solution for the wrapURI case?

Reply:
Please remove it here and fix it in wrapUri